tests/theia_automation_lsp/lib/common_engine.py

```diff
@@ -14,8 +14,7 @@
 import time
 from time import sleep
 from robot.api.deco import keyword
-from selenium.common.exceptions import WebDriverException, NoSuchElementException, TimeoutException, \
-    StaleElementReferenceException
+from selenium.common.exceptions import WebDriverException, NoSuchElementException, StaleElementReferenceException
```

Comment:
This file have ~900 lines... since we introduced new elements, is possible plan an effort to split this file in smaller chunks? We can create a separate TD..

Reply:
I've created a separate issue to track these changes #337